OsmAnd/src/net/osmand/plus/download/DownloadActivity.java

```diff
@@ -708,7 +708,7 @@ public void initAppStatusVariables() {
 
 	}
 
-	public static class AskMapDownloadFragment extends BottomSheetDialogFragment {
+	public static class AskMapDownloadFragment extends OsmAndSheetDialogFragment {
```

Comment:
I suggest to make getSheetDialogType abstract in OsmAndSheetDialogFragment and create another abstract class BottomSheetDialogFragment on top of OsmAndSheetDialogFragment, so we don't need to change source code in all these inheritors. Also it would be logical to create TopSheetDialogFragment.